### ISSUE TITLE
Update MIME type for woff

### ISSUE THREAD

**Issue by vishsid73** [MaxCDN/bootstrap-cdn#494]
Hi,

I am getting this issue

Resource interpreted as Font but transferred with MIME type application/x-woff: "https://netdna.bootstrapcdn.com/font-awesome/3.0.2/font/fontawesome-webfont.woff?v=3.0.2"

I am getting Content-Type →application/x-woff

Please resolve for 3.0.2 version.

Issue Created at FortAwesome/Font-Awesome : FortAwesome/Font-Awesome
https://github.com/FortAwesome/Font-Awesome/issues/5372

**Comment by jmervine** [comment]
Based on this: http://stackoverflow.com/questions/3594823/mime-type-for-woff-fonts

I will change it to `application/font-woff`, unless there are any objections?

**Comment by jmervine** [comment]
Actually @jimaek, this is right in bootstrapcdn.com's Nginx `mime.types` file. This has to be an origin issue.

See: https://github.com/MaxCDN/bootstrap-cdn/blob/develop/mime.types#L84

Also, I verified on the origin that the correct `nginx.conf` and `mime.types` files are being used.

```
[XXXXX@XXXXX ~]$ ps aux | grep nginx.conf
502       2395  0.0  0.0   6372   664 pts/0    S+   19:04   0:00 grep nginx.conf
root     28174  0.0  0.1  56952  2404 ?        Ss    2014   0:00 nginx: master process /usr/local/nginx/sbin/nginx -c /home/XXXXX/bootstrap-cdn/nginx.conf
[XXXXX@XXXXX ~]$ cat bootstrap-cdn/nginx.conf |  grep mime.types
    include       /home/XXXXX/bootstrap-cdn/mime.types;
[XXXXX@XXXXX ~]$ cat bootstrap-cdn/mime.types  | grep woff
    application/font-woff         woff;
```

Curl's
```
bash-3.2$ curl -si "https://netdna.bootstrapcdn.com/font-awesome/3.0.2/font/fontawesome-webfont.woff?v=3.0.2" | head -n15 | grep Content-Type
Content-Type: application/x-woff
bash-3.2$ curl -si "https://maxcdn.bootstrapcdn.com/font-awesome/3.0.2/font/fontawesome-webfont.woff?v=3.0.2" | head -n15 | grep Content-Type
Content-Type: application/x-woff
```

**Comment by jimaek** [comment]
@jmervine I have no access to Bootstrap's Origin unfortunately.

**Comment by XhmikosR** [comment]
@jmervine: why don't you try implementing html5 boilerplate's nginx conf as per #362? It should fix most if not all the server issues.

**Comment by jmervine** [comment]
@XhmikosR The version of nginx being run is a streamlined custom compile and nginx is configured to match. As @jimaek mentioned, it's not an Nginx issue. Admittedly I haven't combed through the html5 boilerplate, but as far as Nginx is concerned, I tend to prefer to roll my own, both binaries and configuration.

**Comment by XhmikosR** [comment]
Suit yourself, but that it's a well tested boilerplate.

**Comment by jmervine** [comment]
I've started going though the many "woff" fonts across the board, updating them to use "application/font-woff", but then noticed that S3's default mime type for .woff is "application/octet-stream". Which means, "woff" is an unrecognized extension per http://docs.aws.amazon.com/AWSImportExport/latest/DG/FileExtensiontoMimeTypes.html. 

As such, I'm going to work with the MaxCDN guys and recommend moving off s3 as our origin, in the mean time, I've updated the above path and a few others (although the cache still needs to be purged at the time of writing this), but I'm going to hold off on the rest.
